po/ja.po

```diff
@@ -1,24 +1,16 @@
 msgid ""
 msgstr ""
-"Content-Type: text/plain; charset=UTF-8\n"
-"Content-Transfer-Encoding: 8bit\n"
 "Project-Id-Version: Copay\n"
-"Plural-Forms: nplurals=1; plural=0;\n"
 "POT-Creation-Date: \n"
 "PO-Revision-Date: \n"
 "Last-Translator: \n"
 "Language-Team: dabura667\n"
+"Language: ja_JP\n"
```

Comment:
Could you use generic language ISO-639-1? I mean, not country specific, just the language. 
In your case, just `Language: ja`. We are not supporting localized languages yet.